quickjs.c

```diff
@@ -38057,10 +38057,10 @@ static JSValue js_error_set_stackTraceLimit(JSContext *ctx, JSValue this_val, JS
 {
     if (JS_IsUndefined(this_val) || JS_IsNull(this_val))
         return JS_ThrowTypeErrorNotAnObject(ctx);
-    int limit;
-    if (JS_ToInt32(ctx, &limit, value) < 0)
+    double limit = 0;
+    if (JS_ToFloat64(ctx, &limit, value) < 0)
         return JS_EXCEPTION;
-    ctx->error_stack_trace_limit = limit;
+    ctx->error_stack_trace_limit = isinf(limit) ? INT_MAX : (int) limit;
```

Comment:
Do we need to handle -Infinity?

Reply:
I don't think we do, it works without it.